src/content/docs/browser/new-relic-browser/installation/update-browser-agent.mdx

```diff
@@ -10,6 +10,7 @@ redirects:
   - /docs/browser/new-relic-browser/installation-configuration/upgrading-browser-agent
   - /docs/browser/new-relic-browser/installation-configuration/upgrade-browser-agent
   - /docs/browser/new-relic-browser/installation/upgrade-browser-agent
+  - /docs/browser/new-relic-browser/installation/update-browser-agent/embed
```

Comment:
any `/embed` pages needn't be redirected. those were used for a since defunct service (docs library service) + nerdlet (nr1-help-xp). they were never intended for other use cases.